vtex/faststore
# fix sonarqube pipeline

## What's the purpose of this pull request?
Our Sonarqube pipeline is triggering some errors and warnings about Typescript ESLint issues, those changes will fix them.

## How to test it?
Those pipelines happens in our store projects. So, we can check if the pipeline will fail in the PRs below.

[Store Components](https://github.com/vtex-sites/storecomponents.store/pull/426)
[Marin Brasil](https://github.com/vtex-sites/marinbrasil.store/pull/289)
